inventory-service/build.gradle

```diff
@@ -26,8 +26,41 @@ dependencies {
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
     testImplementation 'de.flapdoodle.embed:de.flapdoodle.embed.mongo'
     testImplementation 'io.projectreactor:reactor-test'
+    implementation 'org.springframework.boot:spring-boot-starter-validation'
 }
 
 jacoco {
     toolVersion = "0.8.8"
 }
+/**
```

Comment:
Please elaborate as to why you have removed the jacoco testing from this new micro-service

Reply:
because the testing is not implemented yet